appcircle_ios_test_report/1.0.2/component.yaml

```diff
@@ -0,0 +1,41 @@
+platform: iOS
+buildPlatform:
+displayName: Test Reports
+description: Parse test and coverage results.
+docsUrl: https://docs.appcircle.io/continuous-testing/running-ios-unit-and-ui-tests#generating-test-report
+requiredComponents: "appcircle_ios_test"
```

Comment:
@boztopuz If there are any other steps that create a test report, they should also be added here.

Reply:
on the ios side only appcircle_ios_test component generates parsable test report both unit an UI. Other steps releated test does not generate .xcresult file.